docs/user-guide/measure.md

```diff
@@ -75,3 +73,43 @@ Once a measure is drawn, it is possible to add it as an [Annotation](annotations
 <img src="../img/measure/add-as-annotation-ex.jpg" class="ms-docimage"/>
 
 From this step the creation process is the same described in the [Annotations section](annotations.md#add-new-annotation).
+
+## Measurement on the 3D navigation
+
+Thanks to the new improvements made to the **Measure tool**, when [3D Navigation](side-bar.md#3d-navigation) is enabled, the editor has the ability to perform distance, area, point coordinates, height from terrain, angle and slope measurements on the 3D map.
```

Comment:
When the [3D Navigation](side-bar.md#3d-navigation) is enabled, the editor has the ability to perform distance, area, point coordinates, height from terrain, angle and slope measurements on the 3D map.